runtime/src/dpe_platform.rs

```diff
@@ -49,13 +50,15 @@ impl<'a> DpePlatform<'a> {
         cert_chain: &'a ArrayVec<u8, 4096>,
         not_before: &'a NotBefore,
         not_after: &'a NotAfter,
+        dmtf_device_info: &'a [u8],
```

Comment:
nit: Should this be an `Option<...>`. Might make it easier to convey that it may not be popualted.

Reply:
Done